docs/common-issues.md

```diff
@@ -0,0 +1,101 @@
+# Intro
+
+While reviewing pull requests on `zulip-gci` and submissions in the GCI app, and
+when helping students on Zulip, I've encountered some common problems.
+
+The students are learning a lot and they are eager to tackle the challenges.
+That's why I'm always positive:
+
+  * I thank the student for their contribution
+
+    `Thanks for posting the pull request! Looks great.`
+
+  * I point out the issues in their submission
+
+  * I ask them to fix the issues and thank them
+
+    ```
+    Could you submit your task in the app?
+
+    Thanks!
+    ```
+
+This document is a short list of common issues and some ideas on how to address
+them.
+
+## GitHub
+
+### Student posts a PR without submitting the task
+
+I ask the student if they have claimed and submitted the task:
+
+`This looks great! Have you claimed and submitted the task in the GCI app?`
+
+Some students say they'll do it after their PR is merged - I ask them to submit
+the task in the GCI app, with more details:
+
+```
+You should submit the task in the GCI app, so the mentors know what they should
+review - the flow of reviewing the tasks is first claiming them in the app, then
+submitting and waiting for mentor review, then resubmitting them should there be
+a need for fixes.
+
+Could you submit your task in the app?
+```
+
+### Student opens another PR on the same task
+
+I go to all the previous PRs on this task and close them with comment:
+
+`Another PR for this was opened at <PR_hash>`
+
+I also ask the student to learn how to review and edit their commit history:
+
+```
+You don't have to open a new pull request each time you want to change what
+you've submitted - for future reference you can use
+[this guide](https://github.com/zulip/zulip-gci/blob/master/docs/fixing-commits.md)
+for fixing your commits.
```

Comment:
I'd add something here like "Changes pushed to your feature's branch will be automatically updated in the pull request, so you don't have to open a new one every time...".

During the GCI, I spoke to students who knew how to modify commits, but didn't know they would be updated in the PR.